src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasePermissionsController.cs

```diff
@@ -0,0 +1,62 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api
+{
+    [Route("api")]
+    [ApiController]
+    [Authorize]
+    public class ReleasePermissionsController : ControllerBase
+    {
+        private readonly IReleasePermissionService _releasePermissionService;
+
+        public ReleasePermissionsController(
+            IReleasePermissionService releasePermissionService)
+        {
+            _releasePermissionService = releasePermissionService;
+        }
+
+        [HttpGet("publications/{publicationId}/contributors")]
+        public async Task<ActionResult<List<ContributorViewModel>>> GetReleaseContributors(
+            Guid publicationId,
+            [FromQuery] Guid releaseId)
+        {
+            return await _releasePermissionService
+                .GetReleaseContributorPermissions(publicationId, releaseId)
+                .HandleFailuresOrOk();
+        }
+
+        [HttpGet("releases/{releaseId}/contributors")]
```

Comment:
This endpoint should be `publications/{publicationId}/contributors` if we're intending on getting a publication's contributors.